torch/csrc/jit/runtime/profiling_record.cpp

```diff
@@ -145,16 +146,30 @@ void ProfilingRecord::insertShapeProfile(Node* n, Value* i) {
   n->replaceInputWith(i, pn->output());
 }
 
+bool isProfilable(Node* n) {
+  if (tensorexpr::isSupported(n)) {
+    return true;
+  }
+
+  switch (n->kind()) {
+    case prim::Return:
```

Comment:
We shouldn't be profiling `Param` `GetAttr` or `Return` if it doesn't get us anything